challenge3/webapp/y-shinada/app/services/highchart/index_service.rb

```diff
@@ -0,0 +1,47 @@
+class Highchart::IndexService
+  attr_reader :form
+
+  def initialize(form)
+    @form = form
+  end
+
+  def execute
+    house_ids = House.with_city(form.cap_city).
+                  with_num_of_people(form.num_of_people).
+                  pluck(:id)
```

Comment:
```suggestion
                  ids
```
Active Record には `ids` というメソッドがある。
https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-ids